tests/test_llm_special.py

```diff
@@ -143,6 +144,7 @@ def test_llm_command_with_prompt(mock_sql_using_llm, mock_ensure_template, mock_
     mock_sql_using_llm.assert_called()
     assert context == ""
     assert sql == "SELECT 1;"
+    assert isinstance(duration, float)
```

Comment:
Hah. That's clever. I typically do `assert duration > 0.0` but this works too. :)